backend/tracim_backend/lib/core/content.py

```diff
@@ -475,26 +480,26 @@ def _is_content_label_free(
             logger.critical(self, critical_error_text)
             return False
 
-    def _is_content_label_free_or_raise(
+    def _is_filename_free_or_raise(
```

Comment:
when calling a method called `_is_filename_free_or_raise()`, I expect to get a boolean answer or an exception. This is not what you implemented. By the way, the docstring is wrong, because the method do not return `True` if the filename is available.

So you should return True if available or rename this method to `raise_if_filename_not_free`. If you decide to return True, then the code should be something like:

```
def _is_filename_available_or_raise():
    if self._is_filename_available():
       return True
    raise xxx
```

Writing code like this is easier to read:

- the short (in lines of code) case is the first one and you only read the entire method body if you need to